.github/release_workflow.md

```diff
@@ -0,0 +1,34 @@
+# Release Workflow
+
+This document outlines the release workflow for publishing to PyPI and TestPyPI using GitHub Actions.
+
+## Creating a New Release
+
+To create a new release, follow these steps:
+
+1. **Prepare the Release:**
+   - Create a new branch for the release (i.e. `v24.XX`) from `develop`.
+   - Increment the following:
+         - The version number in the `pyproject.toml` file following CalVer versioning.
+         - The`CHANGELOG.md` version with the changes for the new version.
+   - Open a PR to the `main` branch. Once the PR is merged, proceed to the next step.
+
+2. **Tag the Release:**
+   - Create a new Git tag for the release. For a full release, use a tag like `v24.2`. For a release candidate, use a tag like `v24.2-rc.1`.
```

Comment:
I would suggest removing the hyphen (`-`) between the string and the numeric characters, in lieu of PEP 440 versioning compliance, available here: https://peps.python.org/pep-0440/#pre-releases

```suggestion
   - Create a new Git tag for the release. For a full release, use a tag like `v24.2`. For a release candidate, use a tag like `v24.2rc.1`.
```